app/src/org/commcare/activities/PushNotificationActivity.kt

```diff
@@ -96,4 +97,9 @@ class PushNotificationActivity : AppCompatActivity() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        NotificationPrefs.setNotificationAsRead(this)
```

Comment:
Do we need this given we are seeting this as read even before user comes to this page ?